src/main/scala/mesosphere/marathon/core/base/CrashStrategy.scala

```diff
@@ -29,3 +29,12 @@ case object JvmExitsCrashStrategy extends CrashStrategy {
     Runtime.getRuntime.asyncExit(reason.code)(ExecutionContext.Implicits.global)
   }
 }
+
+/**
+  * For testing purposes only: This crash strategy will *NOT* crash but throw an exception
+  */
+case object ThrowExceptionAndDontCrashStrategy extends CrashStrategy {
```

Comment:
let's put this in src/test then?

Reply:
alongside src/test/scala/mesosphere/marathon/test/TestCrashStrategy.scala

Reply:
Good point. Done!